### PR TITLE
Update INSTALL files

### PR DESCRIPTION
**Pull request by byrnHDF**

### CODE REVIEW

File: release_docs/USING_HDF5_VS.txt [hunk 1]
```diff
@@ -78,18 +78,22 @@ Using Visual Studio 2008 with HDF5 Libraries built with Visual Studio 2008
...
 ========================================================================
 
     3.1 FAQ
 
-    Many other common questions and hints are located online and being updated
-    in the HDF Knowledge Base, please see:
+    Community support forum available, please see:
 
-       https://portal.hdfgroup.org/display/knowledge/HDF+Knowledge+Base
+       https://forum.hdfgroup.org/c/hdf4/21
```
Comment (jhendersonHDF) [inline release_docs/USING_HDF5_VS.txt]: Should be https://forum.hdfgroup.org/c/hdf5/8?

File: release_docs/INSTALL_CMake.txt [hunk 2]
```diff
@@ -46,36 +50,33 @@ Note:
...
-Examples Source package:
-    HDF5Examples-2.0.x-Source.tar.gz
+External compression szip and zlib libraries:
+"Use ZLIB from original location"          "https://github.com/madler/zlib/releases/download/v1.2.13"
+"Use ZLIB from original compressed file"   "zlib-1.3.tar.gz"
 
-Configuration files:
-    HDF5config.cmake
-    HDF5options.cmake
+"Use LIBAEC from original location"        "https://gitlab.dkrz.de/k202009/libaec/-/archive/v1.0.6"
```
Comment (hyoklee) [inline release_docs/INSTALL_CMake.txt]: This link requires login and 2 years old already. Is there an alternative? (E.g., https://github.com/erget/libaec/archive/refs/tags/v0.3.4.tar.gz)

Comment (byrnHDF) [reply release_docs/INSTALL_CMake.txt]: This will be changed to match the same URLs used in hdf4.

File: release_docs/INSTALL_CMake.txt [hunk 3]
```diff
@@ -73,8 +73,8 @@ External compression szip and zlib libraries:
 "Use ZLIB from original location"          "https://github.com/madler/zlib/releases/download/v1.2.13"
 "Use ZLIB from original compressed file"   "zlib-1.3.tar.gz"
 
-"Use LIBAEC from original location"        "https://gitlab.dkrz.de/k202009/libaec/-/archive/v1.0.6"
-"Use LIBAEC from original compressed file" "libaec-v1.0.6.tar.gz"
+"Use LIBAEC from original location"        "https://github.com/MathisRosenhauer/libaec/releases/download/v1.0.6"
```
Comment (hyoklee) [inline release_docs/INSTALL_CMake.txt]: I get "Page not found" error.

**Comment by byrnHDF** [comment]
That's because that is part of the file download URL in that GH releases section
https://github.com/MathisRosenhauer/libaec/releases

________________________________________

**Comment by byrnHDF** [comment]
(LIBAEC_GIT_URL "https://github.com/MathisRosenhauer/libaec.git"
is the path for GIT


I guess the docs should use the full path.

________________________________________

File: config/cmake/README.md.cmake.in [hunk 4]
```diff
@@ -2,31 +2,26 @@
 ------------------------------------------------------------------------------
 
 This directory contains the binary (release) distribution of
-@HDF5_PACKAGE_NAME@ @HDF5_PACKAGE_VERSION_MAJOR@ that was compiled on;
+@HDF5_PACKAGE_NAME@ @HDF5_PACKAGE_VERSION_MAJOR@.@HDF4_PACKAGE_VERSION_MINOR@ that was compiled on;
```
Comment (hyoklee) [inline config/cmake/README.md.cmake.in]: HDF4 -> 5?

File: release_docs/USING_HDF5_CMake.txt [hunk 5]
```diff
@@ -239,11 +234,10 @@ command line and the build folder is created as a sub-folder. Windows should
 adjust the forward slash to double backslashes, except for the HDF_DIR
 environment variable.
 
-NOTE: this file is available at the HDF web site:
-    https://portal.hdfgroup.org/display/support/Building+HDF5+with+CMake
+NOTE: this file is available in the HDF5 repository, for more information see:
+    https://github.com/HDFGroup/hdf5/blob/master/release_notes/USING_CMake_Examples.txt
```
Comment (hyoklee) [inline release_docs/USING_HDF5_CMake.txt]: 404 File not Found

**Review by hyoklee — APPROVED** [review]

**Review by derobins — APPROVED** [review]

**Pull request merged**